include/nbl/video/utilities/CComputeBlit.h

```diff
@@ -75,111 +46,143 @@ class CComputeBlit : public core::IReferenceCounted
 		m_dummySSBO = device->createDeviceLocalGPUBufferOnDedMem(creationParams, 1ull);
```

Comment:
do you use this anywhere?

Reply:
It is used [here](https://github.com/Devsh-Graphics-Programming/Nabla/blob/f2b9d520679c7e19f4efc51e9504096ca2d27829/include/nbl/video/utilities/CComputeBlit.h#L259). But I think this is not required anymore because when `coverageAdjustmentScratchBuffer` is `nullptr` i.e. no coverage adjustment is being done then we use `m_blitDSLayouts[EBT_REGULAR]` which doesn't have a SSBO as its 3rd bindings.

Reply:
ok, YAGNI, you can boot it then

Also wont you ahve a big fat error trying to update a descriptor set binding that doesn't exist?

Reply:
Could've been the case, I don't remember.
I think this is irrelevant now.
Resolved?